File: DentilAdmin/src/main/java/dto/DentistDTO.java
package dto;

public class DentistDTO extends PersonalDTO{
	
	public DentistDTO() {
		super();
	}
	
	public DentistDTO(String id, String name, String surname, String address, String phone, String email, String username, String password, String role_name, String jobStart, String jobEnd) {
		super(id, name, surname, address, phone, email, username, password, role_name, jobStart, jobEnd);
	}
}

File: IS/DentilNew/DentilNew/model/dto/DentistDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using DentilNew.model.dto.common;

namespace DentilNew.model.dto
{
    public class DentistDTO : CommonUserTraits
    {
        private string username;
        private string jobStart;
        private string jobEnd;

        public DentistDTO(string id, string name, string surname, string address, string phone, string email, string username, string jobStart, string jobEnd) : base(id, name, surname, address, phone, email)
        {
            this.username = username;
            this.jobStart = jobStart;
            this.jobEnd = jobEnd;
        }

        public string Username { get { return username; } set { username = value; } }
        public string JobStart { get { return jobStart; } set { jobStart = value; } }
        public string JobEnd { get { return jobEnd; } set { jobEnd = value; } }
    }
}
